### DISCUSSION TITLE
Links to Marketplace Actions do not take you to action's full details

### DISCUSSION THREAD

**Discussion by chris904apps** [community/community#62456] [Actions]
### Select Topic Area

Bug

### Body

I am trying to learn how to use actions and while following the docs it appears that the actions being brought into the side panel to should link you to their full details per the docs and the fact that they are hyperlinks but when i click on them, nothing happens.
I tried this in both Edge and Chrome.
Work around is for me to open up new window and go to marketplace directly vs the side panel.

I am thinking that it is a bug and that when you click on the marketplace action in the panel, the details of that action should be populated in the panel so you can work side by side

Docs page: https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions 
![image](https://github.com/community/community/assets/19228678/a05e953b-f8c7-470b-8666-2866b00faae8)

![image](https://github.com/community/community/assets/19228678/8b536342-0711-471c-85b4-af66ad930c2d)

When I try nothing happens, mouse changes to indicate hyperlink but nothing happens when i click on it.
![image](https://github.com/community/community/assets/19228678/cf0f62b2-b081-445a-b481-3659e61bb639)

**Comment by trodrigues** [comment]
Came to here to report the same issue.

This UI seems to be very broken at the moment. Can reproduce in latest Firefox and Chrome.

**Reply by justenbone** [reply]
Same issue 🫣
